pkg/appmanager/as3Manager.go

```diff
@@ -17,10 +23,79 @@ func (appMgr *Manager) processUserDefinedAS3(template as3Template) {
 
 }
 
-// Story 2
-// Covert AS3 JSON in to Map for further processing
```

Comment:
can you add  more description. Like, Function arguments, return value and conditionals and finally a brief description